docs/core/linux-prerequisites.md

```diff
@@ -351,8 +356,10 @@ To install .NET Core 2.x for supported SUSE Linux Enterprise Server and OpenSUSE
 >[!IMPORTANT]
 > To use .NET Core 2.1 with Visual Studio, you need to [install Visual Studio 2017 15.7 Preview 1 or newer](https://www.visualstudio.com/vs/preview).
 
-* .NET Core Runtime 2.1.0-preview1 [install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/runtime-2.1.0-preview1)
-* .NET Core SDK 2.1.300-preview1 [install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/sdk-2.1.300-preview1)
+* [.NET Core Runtime 2.1.0-preview2 install link](https://www.microsoft.com/net/download/linux-package-manager/opensuse/runtime-2.1.0-preview2)
```

Comment:
this is not following the style of the other links

Reply:
Fixed it! Thanks, @mairaw!